inference/generativeai/llm-workshop/lab7-stablelm-base-alpha-7b/stablelm-base-alpha-7b-djl-sagemaker.ipynb

```diff
@@ -0,0 +1,1278 @@
+{
```

Comment:
<p>Picture is missing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3952/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Fixed it in latest commit